service-stacks/private-subnet-public-loadbalancer.yml

```
@@ -80,6 +80,17 @@ Resources:
           Image: !Ref 'ImageUrl'
           PortMappings:
             - ContainerPort: !Ref 'ContainerPort'
+          LogConfiguration:
+            LogDriver: awslogs
+            Options:
+              awslogs-group: !Ref 'AWS::StackName'
+              awslogs-region: !Ref AWS::Region
+              awslogs-stream-prefix: !Ref 'ServiceName'
+
+  LogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Ref 'AWS::StackName'
```

@uhinze Id recommend adding some default retention time here. It can get expensive quick 🤑.
```suggestion
      LogGroupName: !Ref 'AWS::StackName'
      RetentionInDays: 60
```